include/aws/http/http2_stream_manager.h

```diff
@@ -56,20 +56,28 @@ struct aws_http2_stream_manager_options {
     struct aws_byte_cursor host;
     uint16_t port;
 
+    /**
+     * Optional.
+     * HTTP/2 connection configuration. Check `struct aws_http2_connection_options` for details of each config.
+     * Notes for window control:
+     * - By default, client will will maintain its flow-control windows such that no back-pressure is applied and data
+     * arrives as fast as possible.
+     * - For connection level window control, `conn_manual_window_management` will enable manual control. The
+     * inital window size is not controlable.
+     * - For stream level window control, `enable_read_back_pressure` will enable manual control. The initail window
+     * size needs to be set through `initial_settings_array`.
+     */
+    struct aws_http2_setting *initial_settings_array;
+    size_t num_initial_settings;
+    size_t max_closed_streams;
+    bool conn_manual_window_management;
+
     /**
      * HTTP/2 Stream window control.
      * If set to true, the read back pressure mechanism will be enabled for streams created.
      * The initial window size can be set through `initial window size`
```

Comment:
trivial: "The initial window size can be set through `initial window size`" not accurate anymore